tests/providers/test_generic_provider.py

```diff
@@ -21,7 +21,7 @@
     },
     {  # p1: date and time
         'pubdate': '2017-05-18 15:00:15',
-        'expected': datetime(2017, 5, 18, 15, 0, 15, tzinfo=tz.gettz('UTC'))
+        'expected': datetime(2017, 5, 18, 15, 0, 15, tzinfo=tz.tzlocal())
```

Comment:
@medariox this is where I got really confused. Why would we want to use a default timezone of UTC, if we can also use the users, local timezone?